### ISSUE TITLE
FR: 'Insert link to file' should have the same dialog as 'Insert any file'

### ISSUE THREAD

**Issue by larhyp** [zsviczian/obsidian-excalidraw-plugin#1658]
**Is your feature request related to a problem? Please describe.**

'Insert link to file' is working inconsistently compared to 'Insert any file'. 'Insert any file' is giving a possibility in the dialog to point to a section (sub header) in the markdown file, but this is not the case with 'Insert link to a file'. They could possibly have the same dialog available, so you would not have to edit the link manually to point to a certain section of a note.

**Describe the solution you'd like**

Same dialog available in both functions.

**Comment by coneos** [comment]
Yes that would be great not to have to first insert any file to get the link of the section and then pasting this link to the object
